server/admin_management/api/app/routers/router_application_admin.py

```diff
@@ -103,7 +103,7 @@ def create_application_admin(
         )
 
         response = application_admin_service.create_application_admin(
-            application_admin_request, target_user, requester.cognito_user_id
```

Comment:
This looks wrong - create_application_admin above was changed to accept cognito_user_id as an argument?

Reply:
Or this is fine because the service was changed as well.

Reply:
Yes, this line was only a refactored code to accept the 'requester' object change from the service method.